src/routes/FriendList.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import user from '../data/user.json';
+import Friends from '../components/FriendList/Friends';
+import UnderNavBar from '../components/layout/UnderNavBar';
+import { HeaderContains, HeaderText } from '../components/layout/CommonStyle';
+import SearchUser from '../components/FriendList/SearchUser';
+import { useState } from 'react';
+import { AiOutlineUser } from 'react-icons/ai';
+
+const FriendList = () => {
+  const [searchedUser, setsearchedUser] = useState(user);
+  const filteredUser = (searchText: string) => {
+    if (!searchText) {
+      setsearchedUser(user);
+    } else {
+      const filtered = user.filter((element) => {
+        return element.userName
+          .toUpperCase()
+          .includes(searchText.toUpperCase());
+      });
+      setsearchedUser(filtered);
+    }
+  };
+  return (
+    <FriendListContainer>
+      <HeaderContains>
+        <AiOutlineUser size={20} />
+        <HeaderText>Friends</HeaderText>
+      </HeaderContains>
+      <Content>
+        <SearchUser filteredUser={filteredUser} />
+        {searchedUser.map((user) => (
+          <Friends
+            key={user.userId}
+            userProfile={user.userProfile}
+            userName={user.userName}
+            userStatus={user.userStatus}
+          />
+        ))}
+      </Content>
+      <UnderNavBar />
+    </FriendListContainer>
```

Comment:
이 부분도 `<FriendListContainer>`가 단순히 감싸는 용도라면 fragment를 사용하는 게 더 좋을 것 같습니다!
```suggestion
    <>
      <HeaderContains>
        <AiOutlineUser size={20} />
        <HeaderText>Friends</HeaderText>
      </HeaderContains>
      <Content>
        <SearchUser filteredUser={filteredUser} />
        {searchedUser.map((user) => (
          <Friends
            key={user.userId}
            userProfile={user.userProfile}
            userName={user.userName}
            userStatus={user.userStatus}
          />
        ))}
      </Content>
      <UnderNavBar />
    </>
```